.github/workflows/main.yml

```diff
@@ -1,7 +1,9 @@
 ---
 name: CrateDB SQL
 on:
-  pull_request: ~
+  pull_request:
+    paths:
+      - '**.java'
```

Comment:
is this correct? what about changes in gradle files, dependencies, etc?

Reply:
It wasn't. Added gradle as well

Reply:
I'd say that it's not enough, what about `version.properties`, or changes in files under `resources`?

Reply:
Okay, added `version.properties` too. I don't think it's worth making this more complicated than that. The intention is to make this work for the common cases, we still have Jenkins which always runs the full test suite anyways.